source/deployment/saml-faq.rst

```diff
@@ -0,0 +1,18 @@
+Bind Authentication to ID Attribute instead of Email
+------------------------------------------
+
+Alternatively, you can choose to use an Id Attribute instead of email to bind the user.  We recommend chosing an ID that is unique and will not change over time.  
+
+Configuring this will allow you to reuse an email address for a new user without the old user's information being exposed. For instance, if a user with an email address joe.smith@mattermost.com was once an employee, a new employee named Joe Smith can use the same email. This configuration is also useful when a user's name changes and their email needs to be updated. 
+
+This process was designed for backwards compatibility to email binding. Here is a more detailed explanation of the process that will be applied to any new accounts added or accounts logging in after the configuration is set up:  
```

Comment:
I think `was designed with backwards compatibility` is grammatically more accurate, but I'm not 100% sure.

Reply:
Propose

```
Here is a more detailed explanation of the process that will be applied to any new accounts added or accounts logging in after the configuration is set up
```

to

```
Here is the process applied to new account creations and to accounts logging in after the configuration:
```

Reply:
Updated to both proposed edits.